source/whats-new.txt

```diff
@@ -32,6 +33,35 @@ Learn what's new in:
 * :ref:`Version 6.1 <version-6.1>`
 * :ref:`Version 6.0 <version-6.0>`
 
+.. _version-6.12:
+
+What's New in 6.12
+------------------
+
+The {+driver-short+} v6.12 release includes the following features:
+
+- Adds support for v2.0 of the ``zstd`` compression algorithm. For more information about
+  ``zstd`` v2.0, see the ``zstd``
+  :github:`Release Notes <mongodb-js/zstd/releases/tag/v2.0.0>` on GitHub.
+
+- Upgrades the ``bson`` package to v6.10.1. For more information about this release,
+  see the :github:`v6.10.1 Release Notes <mongodb/js-bson/releases/tag/v6.10.1>` for the
+  ``bson`` package on GitHub.
+
+- The driver populates the ``ServerDescription.error`` field when marking a primary
+  replica-set member stale.
```

Comment:
This field was always populated but it is now a new more specific error type and message